.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
       files: [
         'doc/api/esm.md',
         'doc/api/modules.md',
+        'doc/api/modules_module.md',
```

Comment:
Perhaps just calling it `module.md` would be in line with the existing core convention?

Or is that too confusing?

Reply:
No strong opinion from my side, except that changing the url would require me to change all the links 😅 I'll do it if it's the consensus opinion.